clmm/theory/parent_class.py

```diff
@@ -587,7 +591,132 @@ def eval_surface_density_2h(self, r_proj, z_cl, halobias=1., lsteps=500):
         else:
             return self._eval_surface_density_2h(r_proj, z_cl, halobias=halobias, lsteps=lsteps)
 
-    def eval_tangential_shear(self, r_proj, z_cl, z_src, verbose=False):
+    def _get_beta_s_mean(self, z_cl, z_src, z_src_info='discrete', beta_kwargs=None):
+        r"""Get mean value of the geometric lensing efficicency ratio from typical class function.
+
+        Parameters
+        ----------
+        z_cl : float
+            Galaxy cluster redshift
+        z_src : array_like, float, function
+            Information on the background source galaxy redshift(s). Value required depends on
+            `z_src_info` (see below).
+        z_src_info : str, optional
```

Comment:
Same docstring comments as before for the whole new redshift-related docstrings.